### DISCUSSION TITLE
is there any component have dropdown menu with inputbox

### DISCUSSION THREAD

**Discussion by mdnafeed** [primefaces/community#99] [PrimeNG] [answered]
Hii

have any component in primeNG  to create dropdown menu and input box togather ??

![image](https://github.com/primefaces/community/assets/17257897/4e7765e8-e3b8-4ea8-9ca5-46446a006a87)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately, we don't have such a feature yet. I assume you may create a custom component by using inputtext and dropdown together. But logic and styling is up to you.
